packages/cactus-test-tooling/src/main/typescript/fabric/fabric-test-ledger-v1.ts

```diff
@@ -323,6 +370,795 @@ export class FabricTestLedgerV1 implements ITestLedger {
+
+          dataConfigTxGen["Organizations"][0]["Policies"]["Writers"][
+            "Rule"
+          ] = `OR('${mspId}.admin','${mspId}.client')`;
+          dataConfigTxGen["Organizations"][0]["Policies"]["Admins"][
+            "Rule"
+          ] = `OR('${mspId}.admin')`;
+          dataConfigTxGen["Organizations"][0]["Policies"]["Endorsement"][
+            "Rule"
+          ] = `OR('${mspId}.peer')`;
+
+          log.debug("dataConfigTxGen after modificaitons: \n");
+          log.debug(dataConfigTxGen);
+
+          log.debug("dataConfigTxGen after modificaitons: \n");
+          log.debug(dataConfigTxGen);
+
+          const dumpConfigTxGen = yaml.dump(dataConfigTxGen, {
+            flowLevel: -1,
+            quotingType: '"',
+            styles: {
+              "!!int": "decimal",
+              "!!null": "camelcase",
+            },
+          });
+
+          // TODO
+          const regexOrOpening = new RegExp(/OR/g);
+          let replacedConfigTx = dumpConfigTxGen.replace(regexOrOpening, '"OR');
+
+          const regexOrClosing = new RegExp(/\)/g);
+          replacedConfigTx = replacedConfigTx.replace(regexOrClosing, ')"');
+
+          const regexName = new RegExp(/Name:/g);
+          replacedConfigTx = replacedConfigTx.replace(
+            regexName,
+            `&${orgName}\n    Name: `,
+          );
+
+          log.debug(replacedConfigTx);
+
+          // const readersCloseRegEx = new RegExp("')");
+          //dumpConfigTxGen.replace(readersCloseRegEx, "')\"");
+
+          createdFile.filename = `configtx.yaml`;
+          createdFile.filepath = path.join(
+            destinationPath,
+            createdFile.filename,
+          );
+          createdFile.body = replacedConfigTx;
+          log.debug(`Created file at ${createdFile.filepath}`);
+          log.debug(createdFile.filename);
+          await fs.promises.writeFile(createdFile.filepath, createdFile.body);
+
+          return createdFile;
+
+        default:
+          this.log.error(`${fnTag} template type not found`);
+          throw new Error(`${fnTag} template type not found`);
+      }
+    } catch (error) {
+      this.log.error(`populateFile() crashed: `, error);
+      throw new Error(`${fnTag} Unable to run transaction: ${error.message}`);
+    }
+  }
+
+  public setContainer(id: string): void {
+    const fnTag = "FabricTestLedgerV1#setContainer()";
+    let container;
+    try {
+      const docker = new Docker();
+      container = docker.getContainer(id);
+      this.container = container;
+      this.containerId = id;
+    } catch (error) {
+      throw new Error(`${fnTag} cannot set container.`);
+    }
+  }
+
+  public async addExtraOrgs(): Promise<void> {
+    const fnTag = `FabricTestLedger#addOrgX()`;
+    const { log } = this;
+    if (!this.extraOrgs) {
+      throw new Error(`${fnTag}: there are no extra orgs`);
+    }
+
+    log.debug(`
+    Adding ${this.extraOrgs.length} orgs`);
```

Comment:
@RafaelAPB I recommend not putting a new line character inside the logging statement, makes it easier to read later on if all log messages are one line each unless it's an object dump for example.